### PR TITLE
Display speedrun difficulty info

### PR DESCRIPTION
**Pull request by Gheeksy**
Following discussions on my previous PR https://github.com/bvschaik/julius/pull/707

I moved the difficulty information as suggested, and made it visible only if `julius.ini` contains `screen_display_speedrun_infos=1`

Screenshot:
julius_tKEDsVCSxx

### CODE REVIEW

File: src/core/config.c [hunk 1]
```diff
@@ -16,6 +16,7 @@ static const char *ini_keys[] = {
     "gameplay_fix_100y_ghosts",
     "screen_display_scale",
     "screen_cursor_scale",
+    "screen_display_speedrun_infos",
```
Comment (bvschaik) [inline src/core/config.c:19]: The `screen_` prefix is for scaling-related properties. `ui_` is for change to the interface. This change is a UI change, so please rename it to `ui_show_speedrun_info` to be consistent with the other settings. Also note I dropped the "s": "info" is short for "information" which has no plural.

File: src/core/config.h [hunk 2]
```diff
@@ -8,6 +8,7 @@ typedef enum {
     CONFIG_GP_FIX_100_YEAR_GHOSTS,
     CONFIG_SCREEN_DISPLAY_SCALE,
     CONFIG_SCREEN_CURSOR_SCALE,
+    CONFIG_SCREEN_DISPLAY_SPEEDRUN_INFOS,
```
Comment (bvschaik) [inline src/core/config.h:11]: Same naming comment as above: use `CONFIG_UI_SHOW_SPEEDRUN_INFO`.

File: src/window/city.c [hunk 3]
```diff
@@ -92,11 +92,23 @@ static void draw_time_left(void)
     }
 }
 
+static void draw_speedrun_infos(void)
+{
+    if (config_get(CONFIG_SCREEN_DISPLAY_SPEEDRUN_INFOS)) {
+        int x, y, width, height;
+        city_view_get_viewport(&x, &y, &width, &height);
```
Comment (bvschaik) [inline src/window/city.c:99]: `x` is guaranteed to be always 0, and you can use `screen_height()` to get the height.

File: src/window/city.c [hunk 4]
```diff
@@ -92,11 +92,23 @@ static void draw_time_left(void)
     }
 }
 
+static void draw_speedrun_infos(void)
```
Comment (bvschaik) [inline src/window/city.c:95]: Rename to `draw_speedrun_info`.

**Review by bvschaik — COMMENTED** [review]
Looking good! Some minor comments on the naming.

**Comment by Gheeksy** [comment]
Done, thanks for taking the time to explain your comments.
I hope I didn't miss anything.

**Review by bvschaik — APPROVED** [review]
Looks good to me 👍

**Pull request merged**